mappings/net/minecraft/server/PlayerManager.mapping

```diff
@@ -0,0 +1,121 @@
+CLASS net/minecraft/class_166 net/minecraft/server/PlayerManager
+	FIELD field_577 LOGGER Ljava/util/logging/Logger;
+	FIELD field_578 players Ljava/util/List;
+	FIELD field_579 server Lnet/minecraft/server/MinecraftServer;
+	FIELD field_580 chunkMaps [Lnet/minecraft/class_167;
+	FIELD field_581 maxPlayerCount I
+	FIELD field_582 bannedPlayers Ljava/util/Set;
+	FIELD field_583 bannedIps Ljava/util/Set;
+	FIELD field_584 ops Ljava/util/Set;
+	FIELD field_585 whitelist Ljava/util/Set;
+	FIELD field_586 BANNED_PLAYERS_FILE Ljava/io/File;
+	FIELD field_587 BANNED_IPS_FILE Ljava/io/File;
+	FIELD field_588 OPERATORS_FILE Ljava/io/File;
+	FIELD field_589 WHITELIST_FILE Ljava/io/File;
+	FIELD field_590 playerSaveHandler Lnet/minecraft/class_168;
+	FIELD field_591 whitelistEnabled Z
+	METHOD <init> (Lnet/minecraft/server/MinecraftServer;)V
+		ARG 1 server
+	METHOD method_549 getBlockViewDistance ()I
+	METHOD method_550 sendPacketAtPos (DDDDILnet/minecraft/class_169;)V
```

Comment:
[sendToAround](https://maven.fabricmc.net/docs/yarn-1.21+build.6/net/minecraft/server/PlayerManager.html#sendToAround(net.minecraft.entity.player.PlayerEntity,double,double,double,double,net.minecraft.registry.RegistryKey,net.minecraft.network.packet.Packet)) in Yarn